File: go/iq/waffle~get~encryption_metadata+fbid+id_sign+timestamp.go
// BuildWaffleGenerateAccessTokens builds <iq type="get" xmlns="waffle">. Replies with one of: Success, Error.
func BuildWaffleGenerateAccessTokens(smaxId int) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"xmlns": "waffle", "smax_id": smaxId, "to": "s.whatsapp.net", "type": "get"},
		Content: []waBinary.Node{
			{Tag: "encryption_metadata", Attrs: waBinary.Attrs{}},
			{Tag: "timestamp", Attrs: waBinary.Attrs{}},
			{Tag: "fbid", Attrs: waBinary.Attrs{}},
			{Tag: "id_sign", Attrs: waBinary.Attrs{}},
		},
	}
}

File: ts/iq/waffle~get~encryption_metadata+fbid+id_sign+timestamp.ts
/** <iq type="get" xmlns="waffle">. Replies with one of: Success, Error. */
export function buildWaffleGenerateAccessTokens(smaxId: number): BinaryNode {
  return { tag: 'iq', attrs: { xmlns: 'waffle', smax_id: smaxId, to: 's.whatsapp.net', type: 'get' }, content: [
      { tag: 'encryption_metadata', attrs: {  } },
      { tag: 'timestamp', attrs: {  } },
      { tag: 'fbid', attrs: {  } },
      { tag: 'id_sign', attrs: {  } },
    ] }
}